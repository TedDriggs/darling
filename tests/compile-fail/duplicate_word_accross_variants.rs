use darling::FromMeta;

#[derive(Debug, FromMeta, PartialEq, Eq)]
enum Choice {
    #[darling(word)]
    A,
    #[darling(word)]
    B,
    C,
}

fn main() {}
