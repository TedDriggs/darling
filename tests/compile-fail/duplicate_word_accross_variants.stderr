error: Duplicate field `word`
 --> tests/compile-fail/duplicate_word_accross_variants.rs:7:15
  |
7 |     #[darling(word)]
  |               ^^^^
